docs(known-issues): Document that rollbacks are not supported (#340)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/con-relnotes-known-issues.adoc

* Prevent collection of IP addresses by configuring a reverse proxy in front of {product-short}.
//link:{LinkRHIDPIssue}RHIDP-2766[RHIDP-2766]
--

Rollbacks are not supported::
+
--

> If you deploy {product-short} using either Helm or the Operator, bear in mind that rollbacks are not supported.

Reviewer: ```suggestion
If you deploy {product-short} using either Helm or the Operator, rollbacks to earlier versions are not supported.
```
We should specify the platform that we are deploying RHDH on. Is this specific to OCP or also true for other K8s engines?

Author: This affects all platforms, not only OCP.

> It is strongly advised to take regular backups of your {product-short} PostgreSQL database, to be able to restore the application in case of issues.

Reviewer: ```suggestion
Performing regular backups of your {product-short} PostgreSQL database helps you to restore the application to a recent state in case of issues.
```

We avoid language such as "recommended" or "advised" or "strongly encouraged/discouraged". Instead, we simply state the recommended approach and the consequences of taking different approach.

> In the future, we plan to provide guidance on how to perform rollbacks from existing backups.

Reviewer: ```suggestion
We plan to provide guidance on how to perform rollbacks from existing backups in the future
```
Is it ok to "promise" the delivery of future plans here? Maybe we take this out?

Reviewer 2: @linfraze I think it's ok to say "In future versions of {product-short}, we plan..."

Author: I thought that purposely documenting something as a known issue means that we implicitly hope in the future to fix it (unless we document a workaround) since we're aware of it :)

Author: I've rephrased it as `In future versions of {product-short}, ...`

> In the future, we plan to provide guidance on how to perform rollbacks from existing backups.

Author: ```suggestion
In future versions of {product-short}, we plan to provide guidance on how to perform rollbacks from existing backups.
```
